script/manifest/manifestparser.js

```diff
@@ -33,7 +33,7 @@ define('bigscreenplayer/parsers/manifestparser',
          return {
            windowStartTime: windowStartTime,
            windowEndTime: windowEndTime,
-           timeCorrection: timeCorrection
+           correction: timeCorrection
```

Comment:
Any reason for the name change?

Reply:
The strategies do need this to be `correction`. If you refer to the original bigscreenplayer.js line 105, you can see where the return block from this parser is used - timeCorrection is written into time.correction.

Reply:
https://github.com/bbc/bigscreen-player/pull/57/files/a253f11a91b152329b1923f9991409757430a524#diff-d5e95b3ca57b1de771e8a60bafec18b9L105